lib/__tests__/_sendEvent.test.ts

```diff
@@ -41,17 +43,21 @@ describe("sendEvent", () => {
     });
     it("should pass over the payload with multiple events", () => {
       (AlgoliaInsights as any).sendEvent("click", {
-        objectID: "1"
+        eventName: "my-event",
+        indexName: "my-index",
+        objectID: "1",
+        position: 3
       });
       expect(XMLHttpRequest.send).toHaveBeenCalledTimes(1);
       const payload = JSON.parse(XMLHttpRequest.send.mock.calls[0][0]);
       expect(payload).toEqual({
         events: [
           expect.objectContaining({
             eventType: "click",
-            objectID: "1",
-            userID: "42",
-            timestamp: expect.any(Number)
+            eventName: "my-event",
+            indexName: "my-index",
+            objectID: ["1"],
+            position: [3]
```

Comment:
Since the value is an array shouldn't be `positions`?

Reply:
same answer as objectID/objectIDs: https://github.com/algolia/search-insights.js/pull/22/files/b2e374f96fda0c2b903194f879974b41f62618b9#r238710452